### PR TITLE
SC-1063 provide orb that adds support for SonarCloud in CircleCI

### PR DESCRIPTION
**Pull request by tom-vanbraband-sonarsource**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          context: SonarCloud Analysis
+          requires: [orb-tools/publish-dev]
+      - test:
+          name: test-circleci-pyhton
+          image: circleci/python
+          context: SonarCloud Analysis
+          requires: [orb-tools/publish-dev]
+      - orb-tools/publish:
+          context: Publishing Orb
+          orb-ref: sonarsource/sonarcloud@1.0.0
```
Comment (tom-vanbraband-sonarsource) [inline .circleci/config.yml:50]: Version is hard-coded in the circleci config, not ideal

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+          context: Publishing Orb
+          orb-name: sonarsource/sonarcloud
+          requires: [orb-tools/pack]
+      - test:
+          name: test-node
+          image: node
+          context: SonarCloud Analysis
+          requires: [orb-tools/publish-dev]
+      - test:
+          name: test-circleci-pyhton
```
Comment (benoit-sns) [inline .circleci/config.yml]: `pyhton` -> `python`

File: sonar-project.properties [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+sonar.projectVersion=1.0
+ 
+# Path is relative to the sonar-project.properties file. Replace "\" by "/" on Windows.
+# This property is optional if sonar.modules is set. 
+sonar.sources=.
+
+sonar.organization=sonarsource
+ 
+# Encoding of the source code. Default is default system encoding
+#sonar.sourceEncoding=UTF-8
```
Comment (benoit-sns) [inline sonar-project.properties]: I'm nitpicking but we could remove comments

**Review by benoit-sns — COMMENTED** [review]
LGTM

**Pull request closed without merging**